lib/ansible/galaxy/collection/__init__.py

```diff
@@ -246,6 +263,26 @@ def verify_local_collection(
     return result
 
 
+def verify_file_signature(b_manifest_file, b_detached_signature, keyring):
+    gpg_result, rc = run_gpg_verify(b_manifest_file, b_detached_signature, keyring, display)
```

Comment:
What if we assign the context to `rc` on this level of abstraction?
```suggestion
    gpg_result, gpg_verification_failed = run_gpg_verify(b_manifest_file, b_detached_signature, keyring, display)
```

Reply:
How about gpg_verification_rc? Then it's still apparent it's an int.